### DISCUSSION TITLE
Jcenter依赖下载错误

### DISCUSSION THREAD

**Discussion by LovesAsuna** [mamoe/mirai#962] [General]
从2.3.x版本开始我就发现从gradle引入的console就是壳,具体表现为
![L%S(R{G2F@H1K7RGC1SE4XO](https://user-images.githubusercontent.com/32810896/106761674-09b29180-6670-11eb-83a1-5e0adba39e86.png)
然后通过刷新gradle依赖后发现
![CFI40EUPUJ_1CH623WOFN}B](https://user-images.githubusercontent.com/32810896/106761758-1e8f2500-6670-11eb-8b1a-91e8a688b3e8.png)
经过查询发现这个依赖并没有链接到jcenter的公有仓库因此导致了下载错误
![CI90VPEDF1X@S_{@FMDAKX9](https://user-images.githubusercontent.com/32810896/106761913-4088a780-6670-11eb-95d9-9042177ca1f1.png)

**Comment by Him188** [comment]
bintray 放假了? 我发的请求还没收到回复...

**Comment by Him188** [comment]
可以添加仓库 https://dl.bintray.com/him188moe/mirai 临时解决问题

**Reply by LovesAsuna** [reply]
> 可以添加仓库 https://dl.bintray.com/him188moe/mirai 临时解决问题

发这个之前已经按照这个解决了
